### PR TITLE
Seperate ETag from plugin.json

### PR DESCRIPTION
**Pull request by taooceros**

### CODE REVIEW

**Comment by jjw24** [comment]
What's the benefit in separation?

**Comment by taooceros** [comment]
> What's the benefit in separation?

To avoid merge conflict because of etag update.

File: etags.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "54A6672C06E047A48A7D88C78FA5F13E": "W/\"0636966abd1b77a3a42d5edd74c626066713bdea9c1be3127d3116e38e3d66d0\"",
+    "81b8c8c8-72ef-427c-a9c1-17cf8f1c766c": "W/\"490b9590e55144b6005626f67a33538b80584d448f0cc5996a2df843b3c4dfde\"",
+    "F71F6EE367D34BAFB681A9287ACF59EB": "W/\"e9694405abb5ae368a4a2afaa921c726429069e6eed76059c6f6813a9135831b\"",
+    "5B7E53D506844D2D8B7001AA19F5EF8F": "W/\"67550487f7d57ce848fea717fb5e0aa02f3a5c7c2f0d7cc75e03d36cccac543f\"",
+    "1b7c732781f94b748ed08351f8200894": "W/\"12590782b6f4daf8f83e3c764ee7958810c6992bdfa0eada56673d20d68b6734\"",
+    "6c22cffe6b3546ec9087abe149c4a575": "W/\"d3562260d0a9d035c0d6576d6be19c2fecaab0a6e1c99ce2ad8225e42859d4cb\"",
+    "b9aaddfa36a948f384b31b4fb6186861": "W/\"6ddf7cabaa3150e442c1e2d650c7507e959fcce30330cc1218a1828555e105eb\"",
+    "5403b2dc28fb4327aedc9f8c872d9d1f": "W/\"8f03aee8ff41450976f800e1dc6b5d95752ea5dd2e39e0f5b4992a03117335b4\"",
+    "FF2C31D6A47348FFB9ED4EB26F6794E2": "W/\"9bf9b8a844429b8ed1808b6892dcddf2c19206f2e1ed7c79d3e2626a2fb36a1c\"",
+    "84ddc2a6d43911e8a8d5f2801f1b9fd1": "",
```
Comment (jjw24) [inline etags.json:22]: how come this one doesnt have etag

Comment (jjw24) [reply etags.json:22]: doesnt have one in the main branch either

Comment (Garulf) [reply etags.json:22]: Oh! That plugin is Timestamp from @Zeroto521. It doesn't have a release.

Comment (jjw24) [reply etags.json:22]: it's pointing to your fork though https://github.com/Flow-Launcher/Flow.Launcher.PluginsManifest/blob/4d832f4b467d943a17d4f6fe344a1d4234fc6f33/plugins.json#L276

Comment (jjw24) [reply etags.json:22]: i will merge it in and fix later

**Review by jjw24 — APPROVED** [review]

File: etags.json (hunk 1 above)
Comment (Garulf) [reply etags.json:22]: It looks like updater.py uses the `UrlSourceCode` to retrieve the release and eTag.

Comment (jjw24) [reply etags.json:22]: yeah i figure it would be code related, that's why i merged it in.

**Pull request merged**